### DISCUSSION TITLE
Star trails around polaris

### DISCUSSION THREAD

**Discussion by Girthy95** [Stellarium/stellarium#3666] [Q&A] [answered]
I'm trying show the rotation of the stars around polaris, something like a star trail timelapse. Is this possible?

Something like this:
![star-trails-2](https://github.com/Stellarium/stellarium/assets/111137415/4ef5057a-ed8c-4cf1-b23e-66e678edd964)

Thank you!

**Comment by alex-w** [comment, marked as answer]
Please see https://github.com/Stellarium/stellarium/pull/1888

The feature is not in main package yet.
